### DISCUSSION TITLE
Windows 7, crash on start

### DISCUSSION THREAD

**Discussion by Mnior** [helix-editor/helix#11643] [General]
### Summary

Crash on start, dialog was opened:
---------------------------
hx.exe - Entry Point Not Found
---------------------------
The procedure entry point ProcessPrng could not be located in the dynamic link library bcryptprimitives.dll. 
---------------------------
OK   
---------------------------

I found a discussion and solution to this problem at [rusdesk](https://github.com/rustdesk/rustdesk/discussions/7503) project.

### Reproduction Steps

 

I tried this:

1. Run `hx.exe` on windows 7. Double click, or run in cmd.

I expected this to happen:

A dialog with an error opened.

### Helix log

The log was not created.

### Platform

Windows 7, Version 6.1 (Build 7601, Service Pack 1)

### Terminal Emulator

cmd.exe

### Installation Method

releases page

### Helix Version

24.07 (079f544)

**Comment by chtenb** [comment]
Do you happen to have a virus scanner that shows any events?

**Comment by kirawi** [comment]
If you are suggesting we support Windows 7 releases, then we won't. The Rust toolchain does not guarantee support for EOL OSes.
